src/NUnitFramework/framework/Constraints/ExactCountConstraint.cs

```diff
@@ -102,6 +101,8 @@ public override string Description
                 return _itemConstraint != null ? PrefixConstraint.FormatDescription(descriptionPrefix, _itemConstraint) : descriptionPrefix;
             }
         }
+
+    
```

Comment:
Would you please remove these blank lines? (A habit I've gotten into is to view the diff before I commit so that I don't accidentally add things like this.)

Reply:
:+1:

Reply:
This fix looks like it might have been missed with the rest you did. I'm fan of rebasing/amending/squashing within a PR, so feel free to make your commits all nice and shiny if you feel like it.